docs/release_notes/v1.2.0.md

```diff
@@ -0,0 +1,271 @@
+- **RESOLVED** Add to Actor Conceptual docs [804](https://github.com/dapr/docs/issues/804)
+- **RESOLVED** Create FAQ/Troubleshooting on local mDNS [1173](https://github.com/dapr/docs/issues/1173)
+- **RESOLVED** Actor concepts missing types and ids [1298](https://github.com/dapr/docs/issues/1298)
+- **RESOLVED** Article needed to link to Azure APIM [1303](https://github.com/dapr/docs/issues/1303)
+- **RESOLVED** Outdated instructions for running Kafka on Kubernetes [1327](https://github.com/dapr/docs/issues/1327)
+- **RESOLVED** chore: add documentation on the binding mqtt based on the pubsub mqtt… [1328](https://github.com/dapr/docs/pull/1328)
+- **RESOLVED** [CONTENT] PubSub add more redis client settings [1339](https://github.com/dapr/docs/issues/1339)
+- **RESOLVED** Update HashiCorp Vault secret docs [1354](https://github.com/dapr/docs/issues/1354)
+- **RESOLVED** Update hashicorp-vault.md [1355](https://github.com/dapr/docs/pull/1355)
+- **RESOLVED** Add documentation for PowerShell and Command Prompt [1373](https://github.com/dapr/docs/issues/1373)
+- **RESOLVED** Remove workaround for mTLS bug [1387](https://github.com/dapr/docs/issues/1387)
+- **RESOLVED** [CONTENT] Add CLI reference docs for `--data-file` flags for invoke and publish [1391](https://github.com/dapr/docs/issues/1391)
+- **RESOLVED** Add Alibaba Cloud Nacos binding Doc [1401](https://github.com/dapr/docs/pull/1401)
+- **RESOLVED** Add Alibaba Cloud RocketMQ pubsub Doc [1405](https://github.com/dapr/docs/pull/1405)
+- **RESOLVED** Add Alibaba Cloud RocketMQ binding doc [1406](https://github.com/dapr/docs/pull/1406)
+- **RESOLVED** Add Alibaba Cloud DingTalk Webhook binding doc [1407](https://github.com/dapr/docs/pull/1407)
+- **RESOLVED** [CONTENT] Consul nameresolution component [1408](https://github.com/dapr/docs/issues/1408)
+- **RESOLVED** sentinel middleware doc  [1439](https://github.com/dapr/docs/pull/1439)
+- **RESOLVED** Add documentation for the Zeebe bindings [1441](https://github.com/dapr/docs/pull/1441)
+- **RESOLVED** Allow choosing which APIs are available on a sidecar  [1460](https://github.com/dapr/docs/issues/1460)
+- **RESOLVED** v1.2 Release [1461](https://github.com/dapr/docs/issues/1461)
+- **RESOLVED** Documentation for Preview Features [1463](https://github.com/dapr/docs/issues/1463)
+- **RESOLVED** Document how to set the default values for actors [1470](https://github.com/dapr/docs/issues/1470)
+- **RESOLVED** [CONTENT] Document pubsub without CE for 1.2 [1474](https://github.com/dapr/docs/issues/1474)
+- **RESOLVED** Create a shareable Dapr PowerPoint deck [1477](https://github.com/dapr/docs/issues/1477)
+- **RESOLVED** [CONTENT] Add CLI reference docs for `--dapr-http-max-request-size` flags for run [1481](https://github.com/dapr/docs/issues/1481)
+- **RESOLVED** Add Docs for Actor Reentrancy [1482](https://github.com/dapr/docs/issues/1482)
```

Comment:
Add a linebreak is better, same as below

Reply:
The linebreaks don't change the rendering of the markdown. Even if we add the linebreaks it will look the same.

Reply:
I'll leave these without spaces so that all release changes are continuous block. Makes it easier to identify the changes in the Markdown representation when compared to the other information in this document.